Specs/Sequence/English/QuotedTextModel.json

```diff
@@ -0,0 +1,54 @@
+[
+	{
+		"Input": "This “number” is `123` or `456`",
```

Comment:
What happens for cases like these below?
- Between 2" and 5".
- It should say: "I finished the race under 1'46"8 yesterday."
We should have well defined behaviour for both and add specs to show they work right.

Reply:
I have discussed with Chris, and we think currently these corner cases could be ignored.

Reply:
OK, let's keep the spec and mark as NotSupported.